examples/ignition/bootkube-controller.yaml

```diff
@@ -133,7 +133,7 @@ storage:
           # Wrapper for bootkube start
           set -e
           BOOTKUBE_ACI="${BOOTKUBE_ACI:-quay.io/coreos/bootkube}"
-          BOOTKUBE_VERSION="${BOOTKUBE_VERSION:-v0.2.6}"
+          BOOTKUBE_VERSION="${BOOTKUBE_VERSION:-v0.3.0}"
```

Comment:
Talked IRL. If we could make a4739ed48f09ad75309479660bbe0ef7778c266c into a v0.3.1, those could be aligned and it would also simplify the Tectonic bump to v0.3.1.